modules/getting-started/con-audit-log-file-rotation-overview.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+// assembly-audit-log.adoc
+//
+
+[id="con-audit-log-file-rotation-overview_{context}"]
+= About audit log file rotation
+
+Logging to a rotating file in {product} is helpful for persistent storage of audit logs.
+
+== Default settings
+
+Audit logging to a rotating file is disabled by default. When it is enabled, the default behavior changes to:
+
+* Rotate logs at midnight (local system timezone).
+* Log file format: `redhat-developer-hub-audit-%DATE%.log`.
+* Log files are stored in `/var/log/redhat-developer-hub/audit`.
+* No automatic log file deletion.
+* No gzip compression of archived logs.
+* No file size limit.
+
+Audit logs are written in the `/var/log/redhat-developer-hub/audit` directory.
+
+== Log file names
+
+Audit log file names are in the following format: `redhat-developer-hub-audit-%DATE%.log` where `%DATE%` is the format specified in [`auditLog.rotateFile.dateFormat`](#configuring-the-file-rotation-frequency). You can customize file names when you configure file rotation.
```

Comment:
The cross-reference in this sentence is not rendering properly, as the xref format appears to be incorrect.